shopinvader/wizards/shopinvader_partner_binding_line.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+
+
+class ShopinvaderPartnerBindingLine(models.TransientModel):
+    """
+    Wizard lines used to bind manually some partners on shopinvader backends
+    """
+
+    _name = "shopinvader.partner.binding.line"
+    _description = "Shopinvader partner binding line"
+
+    shopinvader_partner_binding_id = fields.Many2one(
+        comodel_name="shopinvader.partner.binding",
+        string="Wizard",
+        required=True,
+        ondelete="cascade",
+    )
+    partner_id = fields.Many2one(
+        comodel_name="res.partner",
+        string="Partner",
+        required=True,
+        ondelete="cascade",
+        readonly=True,
+    )
+    email = fields.Char(
+        related="partner_id.email", required=True, readonly=True
+    )
+    bind = fields.Boolean(
+        help="Tick to bind the partner to the backend. Untick to unbind it."
+    )
+
+    @api.multi
+    def action_apply(self):
+        """
+        Bind selected partners
+        :return: dict
+        """
+        for record in self.filtered(lambda r: r.bind):
```

Comment:
@acsonefho WE must raise a UserError if bind is unticked since this functionality is not implemented.